src/pretix/plugins/banktransfer/models.py

```diff
@@ -80,3 +86,37 @@ def shred_private_data(self):
     class Meta:
         unique_together = ('event', 'organizer', 'checksum')
         ordering = ('date', 'id')
+
+
+class RefundExport(models.Model):
+    event = models.ForeignKey('pretixbase.Event', related_name='banktransfer_refund_exports', on_delete=models.CASCADE, null=True, blank=True)
+    organizer = models.ForeignKey('pretixbase.Organizer', related_name='banktransfer_refund_exports', on_delete=models.PROTECT, null=True, blank=True)
+    datetime = models.DateTimeField(auto_now_add=True)
+    testmode = models.BooleanField(default=False)
+    rows = models.TextField(default="[]")
+    downloaded = models.BooleanField(default=False)
+
+    @cached_property
+    def entity_slug(self):
+        if self.organizer:
+            return self.organizer.slug
+        else:
+            return self.event.slug
+
+    @cached_property
+    def currency(self):
+        if self.event:
+            return self.event.currency
+        return self.organizer.events.first().currency
+
+    @cached_property
+    def rows_data(self):
+        return json.loads(self.rows)
```

Comment:
A ``@cached_property`` on a model can get really confusing, if you change ``rows``, ``rows_data`` won't change. I think it's fine for the entity_slug and currency since they should never change, but I'd avoid it for the other ones, even if that causes a few more CPU cycles